en/docs/index.md

```diff
@@ -2,4 +2,6 @@
 
 The Visual Studio Code Ballerina extension provides a set of rich language features along with an enhanced user experience. It offers easy development, execution, debugging, and testing for the [Ballerina programming language](https://ballerina.io/). The Ballerina language possesses a bidirectional mapping between its syntaxes and the visual representation. You can visualize the graphical representation of your Ballerina source further via the extension.
 
+For more information on the Ballerina VS Code extension, see the [VS Code marketplace](https://marketplace.visualstudio.com/items?itemName=WSO2.ballerina).
```

Comment:
Do we have more information about the extension in the marketplace? I thought we are bringing all that information into the docs. More information is in the documentation isn't it.

Reply:
Updated.